integration-tests/eks-addon/validateResources_test.go

```diff
@@ -77,10 +71,10 @@ func TestK8s(t *testing.T) {
 		fmt.Println("service name: " + service.Name + " namespace:" + service.Namespace)
 	}
 	assert.Len(t, services.Items, 4)
-	assert.Equal(t, "amazon-cloudwatch-agent", services.Items[0].Name)
-	assert.Equal(t, "amazon-cloudwatch-agent-headless", services.Items[1].Name)
-	assert.Equal(t, "amazon-cloudwatch-agent-monitoring", services.Items[2].Name)
-	assert.Equal(t, "amazon-cloudwatch-agent-operator-webhook-service", services.Items[3].Name)
+	assert.Equal(t, "amazon-cloudwatch-observability-webhook-service", services.Items[0].Name)
+	assert.Equal(t, "cloudwatch-agent", services.Items[1].Name)
```

Comment:
nit: all of these values can be consts so if they change in the future, it will be easier to make modifications

Reply:
+1